README.md

```diff
@@ -64,6 +64,15 @@ known anomalies) that you would like to see incorporated into NAB.
 The NAB version will be updated whenever new data (and corresponding labels) is
 added to the corpus; NAB is currently in v1.0.
 
+#### Additional Scores
+
+Here are the NAB scores for some additional flavors of HTM. HTM detector with tm_cpp uses the implementation of temporal memory in https://github.com/numenta/nupic.core/blob/master/src/nupic/algorithms/TemporalMemory.hpp. HTM detector with no likelihood uses the raw anomaly scores directly.
```

Comment:
Now that we have multiple HTM results, this section should make clear which detector algorithm ("numenta", "numentaTM") corresponds to which results in these tables.